config/routes.rb

```diff
@@ -12,8 +12,10 @@
 
   namespace :student do
     root to: redirect('/auth/google_oauth2'), as: 'auth'
-    get '/class_code', to: 'class_code#index', as: 'class_code'
+    get '/class_code', to: 'class_code#new', as: 'class_code'
     get '/failure', to: 'unregistered#index', as: 'unregistered'
+    post '/survey', to: 'survey#create'
+    get '/survey', to: 'survey#show'
   end
-  get '/auth/google_oauth2/callback', to: 'survey#new'
+  get '/auth/google_oauth2/callback', to: 'student/survey#new'
 end
```

Comment:
I think we're okay right now, but I was wondering if we did want to have any kind of name spacing or nested-resources with our routes. At this point, we're fine.